autogluon/utils/try_import.py

```diff
@@ -48,3 +48,13 @@ def try_import_cv2():
         raise ImportError(
             "Unable to import dependency cv2. "
             "A quick tip is to install via `pip install opencv-python`. ")
+
+def try_import_gluonnlp():
```

Comment:
Because AutoGluon imports virtually all files on import autogluon, this will make autogluon raise ImportError on import autogluon. NLP code either needs to be removed from AutoGluon or all usage of gluonnlp has to move to lazy initialization.

Reply:
I see. Should we revise the import structure accordingly? I find that we have https://github.com/awslabs/autogluon/blob/3c0c51630530a2381ac34383d9da8acbc0a9c580/autogluon/__init__.py#L25

Reply:
Okay... Just noticed that the whole code base use `from xxx import *`